NWNXLib/Services/PerObjectStorage/PerObjectStorage.hpp

```diff
@@ -36,12 +34,12 @@ class PerObjectStorage
 
     static inline char GffFieldName[] = "NWNX_POS";
 
-    static void CNWSObject__CNWSObjectDtor__0_hook(bool, CNWSObject* thisPtr);
-    static void CNWSArea__CNWSAreaDtor__0_hook(bool, CNWSArea* thisPtr);
-    static void CNWSPlayer__EatTURD_hook(bool, CNWSPlayer* thisPtr, CNWSPlayerTURD* pTURD);
-    static void CNWSPlayer__DropTURD_hook(bool, CNWSPlayer* thisPtr);
-    static void CNWSUUID__SaveToGff_hook(bool, CNWSUUID* pThis, CResGFF* pRes, CResStruct* pStruct);
-    static void CNWSUUID__LoadFromGff_hook(bool, CNWSUUID* pThis, CResGFF* pRes, CResStruct* pStruct);
+    static void CNWSObject__CNWSObjectDtor__0_hook(CNWSObject* thisPtr);
+    static void CNWSArea__CNWSAreaDtor__0_hook(CNWSArea* thisPtr);
+    static void CNWSPlayer__EatTURD_hook(CNWSPlayer* thisPtr, CNWSPlayerTURD* pTURD);
+    static void CNWSPlayer__DropTURD_hook(CNWSPlayer* thisPtr);
+    static void CNWSUUID__SaveToGff_hook(CNWSUUID* pThis, CResGFF* pRes, CResStruct* pStruct);
+    static bool CNWSUUID__LoadFromGff_hook(CNWSUUID* pThis, CResGFF* pRes, CResStruct* pStruct);
```

Comment:
bool is not the same as BOOL. BOOL is an int32_t actually

Reply:
It's actually a `bool` in this case: https://github.com/nwnxee/unified/blob/master/NWNXLib/API/API/CNWSUUID.hpp#L29

Reply:
Oh, wow, yeah. That's niv-code if I ever saw one :)